pgactivity/data.py

```diff
@@ -278,13 +278,15 @@ def pg_get_waiting(self, duration_mode: int = 1) -> List[BWProcess]:
         with self.pg_conn.cursor() as cur:
             cur.execute(query, {"min_duration": self.min_duration})
             ret = cur.fetchall()
-        return [BWProcess(**row) for row in ret]
+        return [WaitingProcess(**row) for row in ret]
 
-    def pg_get_blocking(self, duration_mode: int = 1) -> List[BWProcess]:
+    def pg_get_blocking(self, duration_mode: int = 1) -> List[BlockingProcess]:
         """
         Get blocking queries
         """
-        if self.pg_num_version >= 90200:
+        if self.pg_num_version >= 100000:
+            query = queries.get("get_blocking_post_100000")
+        elif self.pg_num_version >= 90200:
             query = queries.get("get_blocking_post_90200")
         elif self.pg_num_version < 90200:
```

Comment:
For clarity, I think we should turn this `elif` into an `else`.

Reply:
You mean the last one ? if so should I fix the other instances of this in data.py ?

Reply:
Yes. If there are more of these, I'd suggest to clean this up in a dedicated commit as this is unrelated to this change.

Reply:
done